### PR TITLE
disfetch: cleanup

### PR DESCRIPTION
**Pull request by figsoda**
###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [21.11 Release Notes (or backporting 21.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2111-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

**Review by q60 — APPROVED** [review]
really appreciate it, couldn't find any time to do this myself. thanks!

**Review by q60 — APPROVED** [review]
:ok_hand:

File: pkgs/tools/misc/disfetch/default.nix [hunk 1]
```diff
@@ -16,14 +14,15 @@ stdenv.mkDerivation rec {
...
+    runHook preInstall
     install -Dm755 -t $out/bin disfetch
+    runHook postInstall
   '';
 
   meta = with lib; {
     description = "Yet another *nix distro fetching program, but less complex";
     homepage = "https://github.com/q60/disfetch";
     license = licenses.mit;
-    platforms = platforms.all;
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/disfetch/default.nix:26]: This should be kept to also build it on darwin.

File: pkgs/tools/misc/disfetch/default.nix [hunk 2]
```diff
@@ -1,6 +1,4 @@
-{ stdenv
-, lib
-, fetchFromGitHub }:
+{ stdenv, lib, fetchFromGitHub }:
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/disfetch/default.nix:1]: That doesn't matter tbh.

Comment (figsoda) [reply pkgs/tools/misc/disfetch/default.nix:1]: yeah but why not

**Review by q60 — APPROVED** [review]
i approve

**Pull request merged**